### PR TITLE
1.9.0 release notes

### PR DESCRIPTION
**Pull request by joejulian**
**What type of PR is this?**

Docs
**What this PR does/ why we need it**:

Release notes for the 1.9.0 release

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,7 +2,30 @@
...
+    * feat: allow additional volumes and volumeMounts
+  * alertmanager:
+    * feat: add volume and volume mounts to spec
+    * feat: add support for serviceAccount.annotations
+  * grafana:
+    * feat: enable adding annotations to all default dashboard configmaps
+    * fix: add missing grafana annotations to k8s-coredns dashboard
+    * fix: reduced CPU utilization and time lag for code_verb:apiserver_request_total:increase30d scrape
+    * fix: invalid image pull policy for the admission webhook patch
+    * fix: alert "KubeNodeUnreachable" no longer fires on an autoscaling scale-down event
```
Comment (samvantran) [inline RELEASE_NOTES.md:27]: this was something i added a few days ago and its a major version chart change. I dont think we should include it in the patch KBA version for 1.9 (that deploys on konvoy 1.4). Was the commit pulled into stable-1.x or is this just accidental release notes added?

Comment (dkoshkin) [reply RELEASE_NOTES.md:27]: Yeah this the commit https://github.com/mesosphere/kubernetes-base-addons/pull/444.
@joejulian I can create a PR to `stable-1.x` with the serviceMonitor changes that is based on the previous revision.

Comment (joejulian) [reply RELEASE_NOTES.md:27]: ok

Comment (samvantran) [reply RELEASE_NOTES.md:27]: yeah let's put this only in KBA 2.x.

Comment (dkoshkin) [reply RELEASE_NOTES.md:27]: https://github.com/mesosphere/kubernetes-base-addons/pull/476

**Pull request merged**